clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -4,9 +4,9 @@
  * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance with the License. You may obtain a
  * copy of the License at
- *
+ * 
```

Comment:
If you are using Eclipse, you can actually configure the Java editor save actions to remove trailing spaces.

This is configured in 'Preferences > Java > Editor > Save Actions'. Check 'Addition actions' and click 'Configure...'. On the 'Code Organizing' tab you can configure the 'Remove trailing whitespace' option.